ipython_parallel/_version.py

```diff
@@ -1,2 +1,9 @@
-version_info = (4, 0, 0, 'dev')
+_version_major = 4
+_version_minor = 0
+_version_patch = 0
+_version_extra = 'dev'
```

Comment:
I would indeed take out the check for version_extra in conf.py and leave _version.py exactly how it is.